lib/data/ledger-state-funded.json

```diff
@@ -0,0 +1,114 @@
+{
+  "personaId": "65ba48f6-591b-437c-ae74-219c8c5d4921",
```

Comment:
Can we always create wallet from the gecko, so that don't have mocked object of state in case we change logic somewhere?

Reply:
Originally was trying to do that. The difference in the UUIDs was giving me problems in consistency during the tests. Let me see what I can do about this.

Reply:
Done.